File: artifacts/hasql/src/MySpace/MusicCatalogue/Statements.hs
-- | Mappings to all queries in the project.
-- 
-- Hasql statements are provided by the 'Hasql.Mapping.IsStatement' typeclass instances indexed by the statement parameter type.
-- 
module MySpace.MusicCatalogue.Statements 
  ( -- ** InsertAlbum
    module MySpace.MusicCatalogue.Statements.InsertAlbum,
    -- ** SelectAlbumByFormat
    module MySpace.MusicCatalogue.Statements.SelectAlbumByFormat,
    -- ** SelectAlbumByName
    module MySpace.MusicCatalogue.Statements.SelectAlbumByName,
    -- ** UpdateAlbumReleased
    module MySpace.MusicCatalogue.Statements.UpdateAlbumReleased,
  )
where

import MySpace.MusicCatalogue.Statements.InsertAlbum
import MySpace.MusicCatalogue.Statements.SelectAlbumByFormat
import MySpace.MusicCatalogue.Statements.SelectAlbumByName
import MySpace.MusicCatalogue.Statements.UpdateAlbumReleased

File: artifacts/rust/src/statements.rs
//! Mappings to all queries in the project.
//!
//! Each sub-module exposes a parameter struct that implements [`crate::mapping::Statement`].

pub mod insert_album;
pub mod insert_multiple_albums;
pub mod select_album_by_format;
pub mod select_album_by_id;
pub mod select_album_by_name;
pub mod select_album_fields;
pub mod select_album_with_filters;
pub mod select_album_with_tracks;
pub mod select_genre_by_artist;
pub mod update_album_recording_returning;
pub mod update_album_released;
